File: Arreglos/Java/Eliminacion.java
package Arreglos.Java;
public class Eliminacion {
    public static void main(String[] args) {
        int [] inputArr = {11, 21, 31, 41, 51, 61};

        System.out.println("Antes de la eliminación, el array es: ");
        for (int j = 0; j < inputArr.length; j++){
            System.out.print(inputArr[j] + " ");
        }

        int[] nuevoArr = new int [5];
        for (int j = 0; j < inputArr.length - 1; j++) {
            nuevoArr[j] = inputArr[j + 1];
        }

        System.out.println("\nDespués de la eliminación, el array es: ");
        for (int j = 0; j < nuevoArr.length; j++){
            System.out.print(nuevoArr[j] + " ");
        }
    }
}

File: Arreglos/C#/Eliminacion.cs
using System;
class program
{
    static void Main(string [] args)
    {
        int [] arr = {11, 21, 31, 41, 51, 61};
        
        Console.WriteLine("El array antes de la eliminación es: ");
        foreach (int elemento in arr)
        {
            Console.Write(elemento + " ");
        }
        Console.WriteLine();

        int []nuevoArr = new int[arr.Length - 1];

        for (int i = 1; i < arr.Length; i++)
        {
            nuevoArr[i - 1] = arr[i];

        }

        Console.WriteLine("Después de la eliminación el array es: ");
        foreach (int elemento in nuevoArr)
        {
            Console.Write(elemento + " ");
        }


    }
}